README.md

```diff
@@ -1,5 +1,15 @@
-# discord.js
+<div align="center">
+  <h1>Discord.js Tutorial</h1>
+  <a href="https://discord.gg/MgVaazZ">
+    <img src="https://discordapp.com/api/guilds/416512197590777857/embed.png" alt="Discord" />
+  </a>
+</div>
 
-Repo for the [DJS turorials](https://www.youtube.com/watch?v=UcLspwognk0&list=PLWnw41ah3I4ZfNLV3by7nB6JO2WcCc3Wj) on [MenuDocs](https://www.youtube.com/channel/UCpGGFqJP9vYvzFudqnQ-6IA)
+Repository for the [Discord.js Turorials](https://www.youtube.com/watch?v=UcLspwognk0&list=PLWnw41ah3I4ZfNLV3by7nB6JO2WcCc3Wj) series on [MenuDocs](https://www.youtube.com/channel/UCpGGFqJP9vYvzFudqnQ-6IA).
 
-[![Discord server](https://discordapp.com/api/guilds/416512197590777857/embed.png?style=banner4)](https://discord.gg/MgVaazZ)
+[![Discord Server](https://discordapp.com/api/guilds/416512197590777857/embed.png?style=banner4)](https://discord.gg/MgVaazZ)
+
+## Links
+- [YouTube](https://www.youtube.com/channel/UCpGGFqJP9vYvzFudqnQ-6IA)
+- [Discord.js Series](https://www.youtube.com/watch?v=UcLspwognk0&list=PLWnw41ah3I4ZfNLV3by7nB6JO2WcCc3Wj)
+- [Discord Server](https://discord.gg/MgVaazZ)
```

Comment:
Why the invite 3 times?

Reply:
With these minor enhancements, just change it to major enhancements, I'm talking links to good resources, links to certain MenuDocs discord.js tutorials etc.